codegen/src/main/java/io/improbable/keanu/codegen/python/VertexProcessor.java

```diff
@@ -43,6 +49,7 @@ static void process(String generatedDir) throws IOException {
         Map<String, Object> root = new HashMap<>();
         List<Import> imports = new ArrayList<>();
         List<PythonConstructor> pythonConstructors = new ArrayList<>();
+        List<String> exportedMethodsList = new ArrayList<>();
```

Comment:
Since we have like ~200 vertices to generate, maybe we should use a `StringJoiner` instead?

```
StringJoiner exportedMethodsJoiner = new StringJoiner(", ");
exportedMethodsJoiner.add("\"" + pythonConstructor.pythonClass + "\"")
...
root.put("exportedMethods", exportedMethodsJoiner.toString())
```